src/core/ext/transport/binder/transport/binder_transport.cc

```diff
@@ -172,6 +172,20 @@ static void cancel_stream_locked(grpc_binder_transport* gbt,
   GRPC_ERROR_UNREF(error);
 }
 
+static bool ContainsAuthorityAndPath(const grpc_binder::Metadata& metadata) {
```

Comment:
There are the relevant changes of this PR. @ctiller can you confirm that this check is indeed required and correct?